app/routes/activities.js

```diff
@@ -108,6 +111,7 @@ module.exports = [
     path: '/activity',
     options: {
       tags: ['api'],
+      auth: { scope: [scopes.admin] },
```

Comment:
Is this not 'Add an activivty' which is available to standard users?

Reply:
Good call, the admin endpoint is `activities`